### DISCUSSION TITLE
Clarification on documentation: useQuery Issues vs successful useAsyncQuery usage

### DISCUSSION THREAD

**Discussion by DidoMarchet** [nuxt-community/apollo-module#527] [General]
Hi!

I've been reading through the documentation, and I see that I can use both `useAsyncQuery` and `useQuery`.

However, when I try to use `useQuery`, I get the following internal server error:

```
Internal server error: Failed to resolve import "@vue/apollo-composable" from "pages/index.vue". Does the file exist?
```

I'm not sure why this is happening, as I've followed the documentation closely.

In the meantime, I've been using `useAsyncQuery` for server-side rendering calls and after user interaction calls (e.g., button clicks) without encountering any issues.

So, my question is: What did I overlook when reading the documentation? How should I be using `useQuery`? Is it more convenient to use `useAsyncQuery` in all situations?

Thanks and kind regards,
Davide

**Comment by lumos94** [comment]
having the same issue here
